src/jvm/test/resources/expected_logs/single_suspension_point.txt

```diff
@@ -17,7 +17,7 @@ The following interleaving leads to the error:
 |                                                                                                                                             | operation(): 0                                                                                                                                        |
 |                                                                                                                                             |   suspendAndCauseFailure() [suspendable: Continuation#1]: 0 at SingleSuspensionPointTraceRepresentationTest.operation(MultipleSuspensionsTests.kt:49) |
 |                                                                                                                                             |     counter ➜ AtomicInteger#1 at SingleSuspensionPointTraceRepresentationTest.suspendAndCauseFailure(MultipleSuspensionsTests.kt:54)                  |
-|                                                                                                                                             |     counter.get(): 1 at SingleSuspensionPointTraceRepresentationTest.suspendAndCauseFailure(MultipleSuspensionsTests.kt:54)                           |
+|                                                                                                                                             |     this$0.counter.get(): 1 at SingleSuspensionPointTraceRepresentationTest.suspendAndCauseFailure(MultipleSuspensionsTests.kt:54)                    |
```

Comment:
Can we avoid `this$0` here?

Reply:
I see no way. I consider the cosmetic changes not urgent as this PR is just an extracted step for native calls.

Reply:
It makes the trace slightly less intuitive. Could you please analyze how to fix it, please? It does not extremely complex.

Reply:
They already existed. Generally, it is an actual legitimate name for outer this parameter in the constructor of an inner class.

Reply:
I see, but I've also noticed you fixed this for some cases. Can your approach be applied to all cases so that all the `this$0` references disappear?

Reply:
I mean this name is absolutely legit and necessary:
```kotlin
class A(val x: Int) {
    inner class B(val x: Int) {
        fun f() = x + this@A.x // x + this$0.x, they must be different not to confuse.
    }
}
```

Reply:
This is not something you type in the code, so I would prefer omitting it in the trace.

Reply:
> This is not something you type in the code

I saw such code many times in my life.

Reply:
How would you like to refer to the outer this?